### ISSUE TITLE
Susi does not remember what the user told it

### ISSUE THREAD

**Issue by Manveer695** [fossasia/susi_skill_data#181]
**Actual Behaviour**
Susi AI does not have a skill to remember what the user told it to.

**Screenshots of the skill in google assistant**
![screenshot_20170624-090713](https://user-images.githubusercontent.com/20307535/27505426-290423d8-58bd-11e7-917d-68842c361f5b.png)

**Would you like to work on the issue?**
@dynamitechetan @jyoth1raditya @madhavrathi @DravitLochan please guide if we could use accounts or memory.json to achieve this skill. Then, lets advance on this skill. Thank you.

**Comment by DravitLochan** [comment]
Good idea! I too proposed about reminders. @Orbiter Should we go for a servlet for reminders similar to memory servlet?

**Comment by Manveer695** [comment]
@DravitLochan As its a milestone now, let's start discussing how this can be achieved. any ideas?

**Comment by Orbiter** [comment]
Thats easy. No server-side and no client-side implementation is needed. If you read the skill tutorial, you will find out!

**Comment by Manveer695** [comment]
@Orbiter Ok. Thanks! I will check to it.

**Comment by Manveer695** [comment]
@Orbiter 
<img width="497" alt="birthdayrememberissue" src="https://user-images.githubusercontent.com/20307535/27723003-69bc3936-5d88-11e7-83d7-ce8f0666b189.PNG">

I need to tackle this type of situation by making a variable named "ali birthday" to store the string "on 20th of feb". If i make a skill using this syntax:
<img width="401" alt="rememberskillbirthdayissue" src="https://user-images.githubusercontent.com/20307535/27723079-cfadd1b4-5d88-11e7-8bfa-9b4d8b4732c0.PNG">

But what is happening right now is the variable made is of the name "ali" and "birthday" is getting ignored. Is it possible to fix this? Thank you.

**Comment by Orbiter** [comment]
You can see the result of the variable assignments in the data object from the json answer. Is the variable set as you expected?

**Comment by Manveer695** [comment]
@Orbiter No sir, it's not set as expected.
As we can see for string "remember that ali birthday is on 20". the variable saved is of the name "_ali" instead of $1$ being "ali birthday" : 
<img width="349" alt="orbitertellissue1" src="https://user-images.githubusercontent.com/20307535/27762946-704b1c5c-5e98-11e7-8254-fc5488a116d5.PNG">

Hence the answer has a "birthday" string as an extra string:
<img width="313" alt="orbitertellissue2" src="https://user-images.githubusercontent.com/20307535/27762950-89944b0c-5e98-11e7-8caa-3d4dc669935c.PNG">
 
Thanks!

**Comment by Orbiter** [comment]
Hm, very special problem but I believe that should be considered as a bug in the variable assignment. Would you like to have a look where the term gets cut at the space?

**Comment by Manveer695** [comment]
@Orbiter I didn't get you sir. Like i want "ali birthday" to be stored as a variable may be like ali_birthday or any other way. How can that be achieved in a skill? Please guide me. Thanks.
